apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/question/WaQuestion.java

```diff
@@ -24,6 +26,7 @@
 
 @NoArgsConstructor
 @Getter
+@Setter
```

Comment:
Issue: We want to avoid `Setter`s on the entities if at all possible. Any change should be handled through a command. This way all business rules can be handled when processing those commands.